src/modules/tools/endstops/Endstops.cpp

```diff
@@ -1200,6 +1200,54 @@ void Endstops::on_gcode_received(void *argument)
                 gcode->stream->printf(" will take effect next home\n");
                 break;
 
+            case 365: //M365, change endstop configs for pipette type
+                float value;
```

Comment:
If you declare variables inside a switch case, the body of the switch case has to be written with braces: 
```case 365: {
// body
} break;
```

In addition, these variables need to be initialized with a value or you need to add a `default` case to the subcode switch, since right now they could be uninitialized if something sends an unhandled subcode.